src/D2L.Bmx/Okta/OktaSessionStorage.cs

```diff
@@ -9,12 +10,31 @@ internal interface IOktaSessionStorage {
 }
 
 internal class OktaSessionStorage : IOktaSessionStorage {
-	void IOktaSessionStorage.SaveSessions( List<OktaSessionCache> sessions ) {
 
+	//Update permission to 600 when a new entry is added
+	private void WriteFile600( string path, string Content ) {
```

Comment:
It's customary to put private methods after public methods.
[This SO answer](https://stackoverflow.com/a/310967/5916310) gives a good overview, and we more or less (but not always) follow that order.

The content of this method may also need to change pending discussion re: my earlier comment.
I'd also suggest a simpler name... `600` in the name seems confusing to me. Like `WriteTextToFile` maybe? I just don't want to emphasize the fact that we're setting file permissions in our code. As I said, I think of it as mostly a "feel good" measure...